lib/transmuxer/transmuxable.rb

```diff
@@ -27,11 +27,17 @@ def transmuxable(unprocessed_file_url)
     end
 
     def transmux
-      job = Transmuxer::Job.new(
+      params = {
         input_url: unprocessed_file_url,
         output_store_path: processed_file_store_path,
         notifications_url: notifications_url
-      )
+      }
+
+      if self.caption_file
```

Comment:
Instead of having a `caption_file` object, I'd rather expect the including AR model to have `caption_file_url` getter defined.